src/main/java/dev/hephaestus/atmosfera/world/context/Hemisphere.java

```diff
@@ -26,17 +25,12 @@ class Hemisphere implements EnvironmentContext {
     private final Map<Identifier, Integer> biomeTags = new ConcurrentHashMap<>();
     private final Map<Biome.Category, Integer> biomeCategories = new ConcurrentHashMap<>();
 
-    private final TagGroup<Block> blockTagGroup;
-    private final TagGroup<Biome> biomeTagGroup;
-
     private int blockCount = 0;
     private int skyVisibility = 0;
 
     Hemisphere(byte[][] offsets, Sphere sphere) {
         this.sphere = sphere;
         this.offsets = offsets;
-        this.blockTagGroup = getPlayer().world.getTagManager().getOrCreateTagGroup(Registry.BLOCK_KEY);
-        this.biomeTagGroup = getPlayer().world.getTagManager().getOrCreateTagGroup(Registry.BIOME_KEY);
```

Comment:
These are now gone, not a big deal since we can access a specific block's or biome's tags through its `RegistryEntry` (see `add()` below), which should actually improve performance too.